backend/Dockerfile

```diff
@@ -13,24 +28,25 @@ RUN aws configure set default.region ap-southeast-1
 
 WORKDIR /usr/home/postmangovsg
 
-COPY shared ../shared
-RUN cd ../shared && npm ci
+# this means npm will only install production deps, and optimizations in the node
+# engine will take place as well
+ENV NODE_ENV=production 
```

Comment:
two purposes for setting this
1. you will only download dependencies and not devDependencies
2. you will run in optimised mode